.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+sudo: false
+addons:
+  apt:
+    packages:
+      - autoconf
+      - automake
+      - build-essential
+      - clang
+      - curl
+      - libboost-dev
+      - libboost-program-options-dev
+      - libogg-dev
+      - ninja-build
+before_install:
+  - curl -L -o patchelf-0.8.tar.gz https://github.com/NixOS/patchelf/archive/0.8.tar.gz
```

Comment:
I checked, and you can grab the tarball from nixos.org via HTTPS: https://nixos.org/releases/patchelf/patchelf-0.8/patchelf-0.8.tar.gz

I believe this would eliminate the need for `autoconf` above and let you remove the `./bootstrap.sh` line below.